CHANGELOG.rst

```diff
@@ -32,6 +32,18 @@ Changelog
 * Using MD5 or SHA1 in :class:`~cryptography.x509.CertificateBuilder` and
   other X.509 builders is deprecated and support will be removed in the next
   version.
+* Added additional APIs exposing
+  :class:`~cryptography.x509.certificate_transparency.SignedCertificateTimestamp` internals, including
+  :attr:`~cryptography.x509.certificate_transparency.SignedCertificateTimestamp.signature_hash_algorithm`,
+  :attr:`~cryptography.x509.certificate_transparency.SignedCertificateTimestamp.signature_algorithm`,
+  :attr:`~cryptography.x509.certificate_transparency.SignedCertificateTimestamp.signature`, and
+  :attr:`~cryptography.x509.certificate_transparency.SignedCertificateTimestamp.extension_bytes`.
+* Added the :class:`~cryptography.x509.certificate_transparency.SignatureAlgorithm`
+  enum, representing valid signature algorithms per :rfc:`6962`.
```

Comment:
We can skip documenting this, it's implicitly documented via  `signature_algorithm` which is the only reason to care.